### PR TITLE
[plugin.video.dailymotion_com] v2.3.4

### PR DESCRIPTION
**Pull request by Gujal00**
### Description
As per #1470 
The current maintainer L0RE has moved on to develop a different addon and this codebase has been left in limbo. As per L0RE, he is happy to handover maintenance of this addon to me.
https://forum.kodi.tv/showthread.php?tid=142626&pid=2656751#pid2656751
The changes include the following:
Make videos playable in Krypton
Cleanup LiveTV section to show only active streams.
Change artwork to reflect current Dailymotion branding
Remove all deprecated options
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.dailymotion_com/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.2.3" provider-name="L0RE">
+<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.3.3" provider-name="gujal">
```
Comment (Rechi) [inline plugin.video.dailymotion_com/addon.xml]: You have to increase version to a higher one then `2.3.3`, otherwise it won't get regenerated.

Comment (Gujal00) [reply plugin.video.dailymotion_com/addon.xml]: Changed as per your advice
Thanks

**Comment by Razzeee** [comment]
Are you able to update the commit message itself or should we handle it on merge?

**Comment by Gujal00** [comment]
Done as advised. Thanks

**Comment by Rechi** [comment]
correct commits message would be [plugin.video.dailymotion_com] v2.3.4

see Checklist which you also checked of

**Comment by Gujal00** [comment]
Sorry, corrected now. Thanks for your patience

**Comment by Razzeee** [comment]
Thanks for your patience and your responsiveness :)

**Comment by Gujal00** [comment]
@Razzeee 
Any further releases that I do with fixes, should I be raising PR for them or will your script pull from my Github.
Please advise. Thanks

**Comment by Razzeee** [comment]
Please do PRs for them. Unfortunatly there is no script in place/developed for that.

**Pull request merged**